connectors/database/postgres.go

```diff
@@ -0,0 +1,39 @@
+package sql
+
+import (
+	"fmt"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+func InitPostgresClient(dbConfig map[string]string) *gorm.DB {
+	host := dbConfig["host-v3"]
```

Comment:
Add validation to check if all required keys (`host-v3`, `username-v3`, `password-v3`, `port-v3`, `database-v3`) exist in the `dbConfig` map before proceeding. This will prevent potential runtime errors due to missing keys. For example, you can iterate over a list of required keys and return an error if any are missing. [important]